src/Compilers/CSharp/Test/Symbol/Symbols/StaticAbstractMembersInInterfacesTests.cs

```diff
@@ -8709,6 +9209,104 @@ class Test
                 );
         }
 
+        [Fact]
+        public void ConsumeAbstractStaticEventAdd_06()
+        {
+            var source1 =
+@"
+public interface I1
+{
+    abstract static event System.Action P01;
+}
+";
+            var source2 =
+@"
+class Test
+{
+    static void M02<T>() where T : I1
+    {
+        T.P01 += null;
+    }
+}
+";
+            var compilation1 = CreateCompilation(source1, options: TestOptions.DebugDll,
+                                                 parseOptions: TestOptions.RegularPreview,
+                                                 targetFramework: TargetFramework.NetCoreApp);
+
+            var compilation2 = CreateCompilation(source2, options: TestOptions.DebugDll,
+                                                 parseOptions: TestOptions.Regular9,
+                                                 targetFramework: TargetFramework.NetCoreApp,
+                                                 references: new[] { compilation1.ToMetadataReference() });
+
+            compilation2.VerifyDiagnostics(
+                // (6,9): error CS8652: The feature 'static abstract members in interfaces' is currently in Preview and *unsupported*. To use Preview features, use the 'preview' language version.
+                //         T.P01 += null;
+                Diagnostic(ErrorCode.ERR_FeatureInPreview, "T").WithArguments("static abstract members in interfaces").WithLocation(6, 9)
+                );
+
+            var compilation3 = CreateCompilation(source2 + source1, options: TestOptions.DebugDll,
+                                                 parseOptions: TestOptions.Regular9,
+                                                 targetFramework: TargetFramework.NetCoreApp);
+
+            compilation3.VerifyDiagnostics(
+                // (6,9): error CS8652: The feature 'static abstract members in interfaces' is currently in Preview and *unsupported*. To use Preview features, use the 'preview' language version.
```

Comment:
It looks like the LangVersion diagnostic for the event access is given whether the accessor is from source or metadata. But for operators (like in `ConsumeAbstractUnaryOperator_06`), the LangVersion diagnostic is only given on the usage if the operator is from metadata. This seems inconsistent.